locales/es.json

```diff
@@ -316,6 +317,7 @@
     "list": "Lista",
     "lists": "Listas",
     "local": "Local",
+    "more_menu": "Otras opciones",
```

Comment:
IMHO, we could translate it like "Más opciones", but also we could handle it inside `es-419.json`. Wdyt?